src/locale/pages/organizeCallAssignment/en.yml

```diff
@@ -9,6 +9,21 @@ blocked:
   subtitle: Targets not ready to be called
   title: Blocked
 callers:
+  actions:
+    customize: Customize queue
+    remove: Remove from assignment
+  add:
+    alreadyAdded: Already added
+    placeholder: Add caller
+  customize:
+    exclude:
+      h: Excluded tags
+      intro: Never call targets with these tags.
+    prioritize:
+      h: Prioritized tags
+      intro: First call targets with these tags.
```

Comment:
Hmm, I'm not sure! I like how the two `intro` texts both have the same "flow" now. But most of all I just want to merge this because I feel sad that I didn't end up doing any work over the holidays. 😞